### PR TITLE
Tags Component and Create Feed Integration

### PR DESCRIPTION
**Pull request by dariusk**
## Description

 * Create a CDS `Chip` component
 * Create a CDS `TagsList` component
   * For adding tags, this component uses `check-ui`'s `MultiSelector` and a `material-ui` `Popover`, just like selecting tags
   * There is a `readOnly` state for the tag list and it must be passed an array of strings (`tags`) and its associated `setTags` function
   * The `maxTags` property displays a `+N` chip where N is the remaining tags not shown over the `maxTags` limit. Hovering the chip shows a tooltip with the remaining tags listed (this is not used in the Create Feed UI but was in the spec for the `TagsList` component)
 * Create a "UI Sandbox" page only visible to superadmins at `/check/me/ui-sandbox`

In the sandbox:
 
image

In the create feed UI:

image

References: CV2-3191

## Type of change

- [ ] Performance improvement and/or refactoring (non-breaking change that keeps existing functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Automated test (add or update automated tests)

## How has this been tested?

Self-tested -- unit tests coming soon!

## Things to pay attention to during code review

I don't believe there is anything majorly tricky here. Worth looking at `Sandbox.js` just to see how the new UI sandbox works as a `QueryRenderer`.

## Checklist

- [X] I have performed a self-review of my own code
- [X] I have commented my code in hard-to-understand areas, if any
- [ ] I have made needed changes to the README
- [X] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [X] If I implemented any new components, they are self-contained, their `propTypes` are declared and they use React Hooks and, if data-fetching is required, they use Relay Modern with fragment containers
- [X] I have removed the /* eslint-disable @calm/react-intl/missing-attribute */ from any files I've worked on and added a `description` prop to all `<FormattedMessage />` components that were missing it.
- [X] To the best of my knowledge, any new styles are applied according to the design system
- [ ] If I added a new external dependency, I included a rationale for doing so and an estimate of the change in bundle size (e.g., checked in https://bundlephobia.com/)

### CODE REVIEW

File: src/app/components/cds/buttons-checkboxes-chips/Chip.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+};
+
+Chip.defaultProps = {
+  removable: false,
+  onRemove: () => {},
+};
+
+Chip.propTypes = {
+  onRemove: PropTypes.func,
+  removable: PropTypes.bool,
```
Comment (amoedoamorim) [inline src/app/components/cds/buttons-checkboxes-chips/Chip.js:40]: This seems a bit redundant. Can we get rid of the removable prop and consider removable if onRemove is passed?

Comment (dariusk) [reply src/app/components/cds/buttons-checkboxes-chips/Chip.js:40]: makes sense, I'll change that

**Review by amoedoamorim — APPROVED** [review]
All good!

File: src/app/components/cds/buttons-checkboxes-chips/Chip.js (hunk 1 above)
Comment (dariusk) [reply src/app/components/cds/buttons-checkboxes-chips/Chip.js]: fixed in [d8dc2cf](https://github.com/meedan/check-web/pull/1481/commits/d8dc2cf87946b8dc67eca4da7ee0ce09e93fbe28)

**Comment by dariusk** [comment]
Same unit tests failing here as on upstream, in `CreateFeed.js`, merging

**Pull request merged**